### PR TITLE
fix: information about persistence for Caching service backup

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Describe your changes

This pull request updates docs to reflect the current behavior of Redis Persistence, specifically regarding the deletion of all existing backups when persistence is switched off.

[DOC-1023](https://aiven.atlassian.net/browse/DOC-1023)

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

[DOC-1023]: https://aiven.atlassian.net/browse/DOC-1023?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/platform/concepts/service_backups.md [hunk 1]
```diff
@@ -270,24 +270,34 @@ To be notified once the PITR feature is available for Cassandra, contact the Aiv
...
+You can configure the `redis_persistence` settings from the **Advanced configuration**
+section on your **Service settings** page in the [Aiven Console](https://console.aiven.io).
+
+- **Enabled (`rdb`)**: When you set `redis_persistence` to `rdb`, Aiven for Caching
+  performs RDB dumps every 10 minutes whenever a key changes. These dumps provide
+  additional data protection against Caching service incidents, limiting potential data
+  loss to a maximum of 10 minutes. However, full backups are created only according to
+  the backup schedule (every 12 hours) or when the service is shut down.
+- **Disabled (`off`)**: When you set `redis_persistence` to `off`, Aiven for Caching
+  stops all Redis RDB dumps and backups If the service restarts or powers off for
```
Comment (bogdanp05) [inline docs/platform/concepts/service_backups.md:287]: `.` after "and backups".

**Review by bogdanp05 — CHANGES_REQUESTED** [review]
The warning looks good.

File: docs/platform/concepts/service_backups.md (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/platform/concepts/service_backups.md]: fixed

**Review by bogdanp05 — APPROVED** [review]

File: docs/platform/concepts/service_backups.md [hunk 2]
```diff
@@ -270,24 +270,34 @@ To be notified once the PITR feature is available for Cassandra, contact the Aiv
...
-  so data can be lost at any moment if the service is restarted for
-  any reason or if the service is powered off. This also means the
-  service can't be forked.
+Aiven for Caching automatically backs up data every 12 hours and supports configurable
+data persistence using Redis Database Backup (RDB).
+
+#### Persistence settings
+
+You can configure the `redis_persistence` settings from the **Advanced configuration**
+section on your **Service settings** page in the [Aiven Console](https://console.aiven.io).
```
Comment (wojcik-dorota) [inline docs/platform/concepts/service_backups.md:279]: nitpicking: you might include the cog icon for `**Service settings**`.

**Review by wojcik-dorota — APPROVED** [review]
One suggestion, otherwise LGTM :-)

**Pull request merged**